containers/images/foreman/Dockerfile

```diff
@@ -32,9 +32,8 @@ COPY container-assets/database.yml /etc/foreman/database.yml
 COPY container-assets/entrypoint.sh /usr/bin/entrypoint.sh
 COPY container-assets/start_foreman.sh /usr/bin/start_foreman.sh
 COPY container-assets/wait_on_postgres.py /usr/bin/wait_on_postgres.py
-COPY container-assets/create_qpid_queue.sh /usr/bin/create_qpid_queue.sh
 
-USER foreman
```

Comment:
Is this change intentional?

Reply:
Yes. It is causing more problems than solving and is not required to run the container. This will be nice to have when we want to push all containers to not have to run as root.